gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/AccountEndpointHandler.java

```diff
@@ -43,9 +55,15 @@
 public class AccountEndpointHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(AccountEndpointHandler.class);
     private final AccountService accountService;
+    private final UserService userService;
+    private final Domain domain;
+    private final PasswordValidator passwordValidator;
 
-    public AccountEndpointHandler(AccountService accountService) {
+    public AccountEndpointHandler(AccountService accountService, UserService userService, Domain domain, PasswordValidator passwordValidator) {
         this.accountService = accountService;
+        this.userService = userService;
+        this.domain = domain;
+        this.passwordValidator = passwordValidator;
```

Comment:
I would prefer to do the resetPassword method in the accountService and avoid all these injections.